GitUI/NBugReports/BugReportInvoker.cs

```diff
@@ -103,7 +103,9 @@ public static void Report(Exception exception, bool isTerminating)
                 LogError(exception, isTerminating);
             }
 
-            if (exception is ExternalOperationException externalOperationException
+            ExternalOperationException externalOperationException = exception as ExternalOperationException;
+
+            if (externalOperationException is not null
                 && externalOperationException.InnerException is { }
                 && externalOperationException.InnerException.Message.Contains(_dubiousOwnershipSecurityConfigString))
```

Comment:
just
```suggestion
            if (externalOperationException?.InnerException?.Message?.Contains(_dubiousOwnershipSecurityConfigString) is true)
```

Reply:
I can't say I'm excited by this syntax but I'll give in.